cluster-autoscaler/cloudprovider/alicloud/alibaba-cloud-sdk-go/sdk/utils/utils.go

```diff
@@ -22,7 +22,7 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"github.com/satori/go.uuid"
+	"github.com/gofrs/uuid"
```

Comment:
should we be using `github.com/google/uuid` instead? For consistency with kubernetes/kubernetes.

Are there any advantages to either?

Reply:
make sense as we already have `github.com/google/uuid` in dependency, I've replaced `gofrs/uuid` with `github.com/google/uuid` using the latest version